### DISCUSSION TITLE
math-concepts.pdf is missing,where i can download this files?

### DISCUSSION THREAD

**Discussion by laniakeawhite** [stack-of-tasks/pinocchio#2114] [Q&A] [answered]
I am learning the concepts about pinocchio,some one have mentioned a useful file:https://stack-of-tasks.github.io/pinocchio/assets/math-concepts.pdf.

It seems the link is broken,does anyone know where can download this file?

**Comment by jcarpent** [comment, marked as answer]
See https://gepettoweb.laas.fr/doc/stack-of-tasks/pinocchio/master/doxygen-html/md_doc_c-maths_se3.html for the math concepts of Pinocchio.
